src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+/**
+ * Duke is a task management application that allows users to manage tasks
+ * by adding, marking as done, deleting, and listing tasks.
+ */
+public class Duke {
+    private final String name;
```

Comment:
I like that you gave the bot a name! It might be useful when dealing with multiple bots.